pkg/util/util.go

```diff
@@ -30,6 +31,7 @@ var (
 	commonDockerRegistries = map[string]bool{
 		"docker.io":                   true,
 		"quay.io":                     true,
+		"k8s.gcr.io":                  true,
```

Comment:
Let's hold onto this change.
The issue here is that we have customers using private registries and storing their images with the complete path. For instance, `private.reg.io/k8s.gcr.io/pause`, etc. 
Until now we used to remove the other common registries from our images, when prefixing to the image. But we missed `k8s.gcr.io` and `gcr.io`. Now adding them here will break the images in existing customer envs.

Reply:
We have a ticket to track this issue - https://portworx.atlassian.net/browse/OPERATOR-240

Reply:
Ok, will remove this